From d2459853e117e104f029a55f305e90629ae23988 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Sun, 23 Sep 2012 15:21:44 +0300
Subject: [PATCH] man/epylint.1: (SEE ALSO): for Cygwin
Organization: Private
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit


Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 man/epylint.1 |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/man/epylint.1 b/man/epylint.1
index f2022e9..3dd6056 100644
--- a/man/epylint.1
+++ b/man/epylint.1
@@ -11,4 +11,4 @@ current package to avoid import error messages. See
 \'s manpage for information about all available options.
 
 .SH "SEE ALSO"
-.B pylint(1), pylint-gui(1)
+.B pylint(1)
-- 
1.7.9

